app/services/eyeshade/publishers.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Eyeshade::Publishers < Eyeshade::BaseApiClient
+  RESOURCE = "/v2/publishers"
+
+  def create_settlement(body: )
+    return {} if Rails.application.secrets[:api_eyeshade_offline]
+
+    response = connection.post do |request|
+      request.headers["Authorization"] = api_authorization_header
+      request.headers["Content-Type"] = "application/json"
+      request.url("#{RESOURCE}/settlement")
+
+      request.body = body.to_json
```

Comment:
Would you write exception handling in case if we get a 503?

Reply:
Sure can, 
https://github.com/brave-intl/publishers/pull/1660/commits/c26b02ec4f3dfe7db967669299103bd36bce630b